config/heroes/tower.json

```diff
@@ -136,10 +142,15 @@
 		[
 			{ "skill" : "wisdom", "level": "advanced" }
 		],
-		"specialties":
-		[
-			{ "type":3, "val": 3, "subtype": 60, "info": 0 }
-		]
+		"specialty" : {
+			"bonuses" : {
+				"hypnotize" : {
+					"subtype" : "spell.hypnotize",
+					"type" : "SPECIAL_SPELL_LEV",
+					"val" : 3
```

Comment:
This bonus is used in `CGHeroInstance::getSpellBonus` and have kind of TIMES_HERO_LEVEL updater there,
```
valOfBonuses(Bonus::SPECIAL_SPELL_LEV, spell->getIndex()) * level
```
 could it be converted?

Reply:
Yes, could move the level out of that formula and add TIMES_HERO_LEVEL updater, but is there a need for such a bonus that doesn't scale with level?

Reply:
It might be useful for modders. I think mostly about unification now. Other bonuses near in code do not have explicit scaling.

Reply:
Same applies to SPECIAL_BLESS_DAMAGE.

Reply:
Done.